jib-core/src/main/java/com/google/cloud/tools/jib/global/JibSystemProperties.java

```diff
@@ -23,6 +23,8 @@
 /** Names of system properties defined/used by Jib. */
 public class JibSystemProperties {
 
+  public static final String _JIB_UPSTREAM_CLIENT = "jib.upstreamClient";
```

Comment:
Ah, I think he meant the variable name should stay as `UPSTREAM_CLIENT`, but the value should be `"_JIB_UPSTREAM_CLIENT"`

Reply:
Ah, yeah. Good catch.

Reply:
oh wov, i wasnt aware you can have "_" in property names.